api/itsecurity-api.yaml

```diff
@@ -766,6 +738,10 @@ components:
       properties:
         comment:
           type: string
+        dummy_not_to_use_field:
```

Comment:
ez legyen sztem ilyen "commentt" vagy hasonló, mintha lenne értelme :D

Reply:
Javítva. :D

Reply:
Meg remélem értékeled, hogy milyen elegánsan kikerültem a bean creation problémát. :DD

Reply:
Azt néztem is, tanultam is belőle, fasza :D